connectors/src/connectors/intercom/lib/help_center_permissions.ts

```diff
@@ -81,6 +85,23 @@ export async function allowSyncHelpCenter({
         websiteTurnedOn: helpCenterOnIntercom.website_turned_on,
         permission: "read",
       });
+
+      // Create datasource folder node
+      const connector = await ConnectorResource.fetchById(connectorId);
+      if (connector !== null) {
+        const dataSourceConfig = dataSourceConfigFromConnector(connector);
+        const helpCenterInternalId = getHelpCenterInternalId(
+          connectorId,
+          helpCenterOnIntercom.id
+        );
+        await upsertDataSourceFolder({
```

Comment:
Same as above, we probably want this in an activity

Reply:
:+1: